target_athena/athena.py

```diff
@@ -30,6 +30,9 @@ def create_client(config, logger: Logger):
     aws_profile = config.get("aws_profile") or os.environ.get("AWS_PROFILE")
     aws_region = config.get("aws_region") or os.environ.get("AWS_REGION")
     s3_staging_dir = config.get("s3_staging_dir") or os.environ.get("S3_STAGING_DIR")
+    athena_workgroup = config.get("athena_workgroup") or os.environ.get(
+        "ATHENA_WORKGROUP"
+    )
```

Comment:
Is it okay to remove the coalesce to the environment variable?

If `--config=ENV` is specified at the command line, the same can be set with the convention `<connector_name>_<setting_name>`, as in `TARGET_ATHENA_ATHENA_WORKGROUP`.

(The `--config=ENV` arg can be used in combination with one or more `config=path/to/config.json` args.)

More info: https://sdk.meltano.com/en/latest/implementation/cli.html#parsing-environment-variables

Reply:
Am I using the --config option with env var incorrectly?
![image](https://user-images.githubusercontent.com/1859449/145401715-a9718618-136b-490a-aefe-9db566a94ff7.png)